### ISSUE TITLE
[Feature Request]: Pojav Launcher support

### ISSUE THREAD

**Issue by Madis0** [Fabulously-Optimized/vanilla-installer#41]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Potential Enhancement

A checkbox users can enable to indicate they want to install to Pojav launcher

### How it could be implemented

The user would download the installer as jar and check the checkbox. The installer would install to assigned path as before, but also unpack a set of patches to specified folders.

### Anything else?

Low priority, get the main thing running first

**Comment by osfanbuff63** [comment]
Just to clarify, we don't need to run this on the Android device running Pojav, this just simplifies the process of getting all the files ready correct? If we're running on Android, that might be a bit harder because I don't know if Qt for Python runs on Android, meaning we're stuck to the CLI (which ofc isn't ideal).

**Comment by Madis0** [comment]
- It needs to be a jar
- It needs to detect the correct path somehow (Fabric installer detected the correct one, but maybe Pojav gave it as a parameter)
- I can do the testing on Pojav itself

**Comment by nsde** [comment]
Shouldn't we create a separate Kotlin/Java Project for Pojav?

**Comment by Madis0** [comment]
Well, I thought you could compile it with something like [Jython](https://www.jython.org). 

But if not, scrap this idea, as it's not worth creating a new project for, then it'd just wait for packwiz's implementation.

**Comment by osfanbuff63** [comment]
> Well, I thought you could compile it with something like [Jython](https://www.jython.org). 
> 
> But if not, scrap this idea, as it's not worth creating a new project for, then it'd just wait for packwiz's implementation.

We could also compile to an APK, allowing us to use the existing VI code base. Not sure if that's idea though.

**Comment by nsde** [comment]
> *The current release (a Jython 2.7.x) only supports Python 2 (sorry)*
> - https://www.jython.org/

Sorry, but I don't think its not a good idea to simply compile it to an APK. It's not as easy as that. Python is just not made for mobile app developement. 

The closest thing to that would be that we could program an `adb` feature, which would install the files via USB. But maybe even that would be worth creating a new repository for, because it's a completely different system.

This repo's name is VanillaInstaller, so it wouldn't make sense to add any unrelated functionality. We should keep it lightweight.

**Comment by osfanbuff63** [comment]
You can compile PySide6 to C++ which can then be compiled to an android APK, allowing this functionality. So I wouldn't rule it out in the future, but not for now.

**Comment by osfanbuff63** [comment]
We could also make this somewhat of a [monorepo](https://en.wikipedia.org/wiki/Monorepo) if we wanted, although I'm not sure.

TL;DR: Maybe at some point, but not high priority right now.

**Comment by nsde** [comment]
> You can compile PySide6 to C++ which can then be compiled to an android APK, allowing this functionality

This is probably way too cumbersome...

**Comment by osfanbuff63** [comment]
> > You can compile PySide6 to C++ which can then be compiled to an android APK, allowing this functionality
> 
> This is probably way too cumbersome...

~~Who knows, it's still an early idea :P~~
Correction, https://pypi.org/project/pyqtdeploy can do this it seems. Again, still an early idea.
